DynamoDB/CreateTable.js

```diff
@@ -0,0 +1,68 @@
+var AWS = require("aws-sdk");
+var keys = require("../keys.json");
+
+
+AWS.config.update({
+  region: keys.region,
+  endpoint: keys.endpoint
+});
+
+var dynamodb = new AWS.DynamoDB();
+
+var params = {
+    TableName : "V-Transfer",
+    KeySchema: [       
+        { AttributeName: "PK", KeyType: "HASH"},  //Partition key
+        { AttributeName: "SK", KeyType: "RANGE" }  //Sort key
+    ],
+    AttributeDefinitions: [       
+        { AttributeName: "PK", AttributeType: "S" },
+        { AttributeName: "SK", AttributeType: "S" },
+        { AttributeName: "u_id", AttributeType: "S" },
+        { AttributeName: "data", AttributeType: "S"}
+    ],
+    ProvisionedThroughput: {       
+        ReadCapacityUnits: 10, 
+        WriteCapacityUnits: 10
+    },
+    GlobalSecondaryIndexes: [{
+        IndexName: "Find_file",
+        KeySchema: [
+            {
+                AttributeName: "u_id",
+                KeyType: "HASH"
+            }
+        ],
+        Projection: {
+            ProjectionType: "ALL"
+        },
+        ProvisionedThroughput: {
+            ReadCapacityUnits: 1,
+            WriteCapacityUnits: 1
+        }
+    }],
+    LocalSecondaryIndexes: [{
+        IndexName: "Filter_by_name",
```

Comment:
suggestion to add a LSI on timestamp attribute to get latest files uploaded by user.

Reply:
any particular reason for that??
As per my knowledge it won't create any difference and frequently used query will be based on timestamp only and adding that into LSI would cause you to query LSI more frequently than global table.